kong/api/routes/certificates.lua

```diff
@@ -1,317 +1,80 @@
-local crud = require "kong.api.crud_helpers"
-local utils = require "kong.tools.utils"
-local cjson = require "cjson"
+local endpoints   = require "kong.api.endpoints"
+local utils       = require "kong.tools.utils"
 
+local function get_cert_by_server_name_or_id(self, db, helpers)
+  local id = self.params.certificates
 
```

Comment:
New line maybe not needed, similar to pattern:
```
local ok, err = func()
if not ok then
end
```

E.g. where test follows immediately the declaration.

Reply:
This is done